src/reflect/scala/reflect/internal/Flags.scala

```diff
@@ -120,7 +120,8 @@ class ModifierFlags {
   final val ARTIFACT      = 1L << 46      // symbol should be ignored when typechecking; will be marked ACC_SYNTHETIC in bytecode
                                           // to see which symbols are marked as ARTIFACT, see scaladocs for FlagValues.ARTIFACT
   final val DEFAULTMETHOD = 1L << 47      // symbol is a java default method
-  final val ENUM          = 1L << 48      // symbol is an enum
+  final val ENUM          = 1L << 48      // symbol is a java enum
+  final val ANNOTATION    = 1L << 49      // symbol is a java annotation
```

Comment:
If I ever find the issue with typechecking Java annotation code correctly, Scala will also be able to emit platform annotations. I think it's not simply a question of Scala-defined vs. Java-defined annotations.